src/platforms/dart/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -21,3 +21,13 @@ The span finishes once the request has been executed. The span `status` depends
 When the HTTP request throws an `Exception`, Sentry's SDK associates this exception to the running span. If you haven't set the SDK to swallow the exception and capture it, the span and SentryEvent will be linked when viewing it on the **Issue Details** page in sentry.io.
 
 For more information see our [SentryHttpClient integration](/platforms/dart/configuration/integrations/http-integration/#performance-monitoring-for-http-requests).
+
+### Dio HTTP Library Instrumentation
+
+The Dio instrumentation, once added the Dio integration feature, starts a span out of the active span bound to the scope for each HTTP Request. The SDK sets the span `operation` to `http.client` and `description` to request `$method $URL`; for example, `GET https://sentry.io`.
```

Comment:
Something is missing in the first part of the first sentence, but I'm not sure what. I've added "to" but that's a guess. I've also rearranged the sentence to improve the flow. 

```suggestion
Once added to the Dio integration feature, the Dio instrumentation starts a span out of the active span bound to the scope for each HTTP request. The SDK sets the span `operation` to `http.client` and the `description` to request `$method $URL`. For example, `GET https://sentry.io`.
```